.circleci/config.yml

```diff
@@ -1,72 +1,72 @@
 defaults: &defaults
-  working_directory: ~/repo
```

Comment:
We use 2 spaces indentation, not sure if this change is due to prettier or to your editor

Reply:
Thank you for pointing out. It may be a smell of prettier misconfiguration. I'll check

Reply:
@gunzip my fault, we forked from [io-template-typescript](https://github.com/pagopa/io-template-typescript) that have a circleci config, initially we didn't know we needed a backend component, now that we know we need one we certainly have to switch to azure pipelines, I've added [this story](https://www.pivotaltracker.com/story/show/175916761)